doc/set_fees_process.md

```diff
@@ -0,0 +1,38 @@
+# Setting fees process
+
+It will be explained on an example of Indy CLI.
+
+### Prerequisites
+
+* libindy
+* Indy CLI
+* libsovtoken
+* Wallet with Trustee DID
+* Created pool in Indy CLI
+
+### Creating a SET_FEES transaction.
```

Comment:
I am feeling we should add a little bit (perhaps links to other documents) about how to get indy-cli running and how to load the libsovtoken plugin using indy-cli.

Reply:
I will try to find such links (but I don't think that they exist)